test/unit/lib/postcode_sanitizer_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class PostcodeSanitizerTest < ActiveSupport::TestCase
+
+  context "postcodes come through to imminence" do
+
+    should "strip trailing spaces from entered postcodes" do
+      assert_equal "WC2B 6NH", PostcodeSanitizer.sanitize("WC2B 6NH ")
+    end
+
+    should "strip non-alphanumerics except underscores from entered postcodes" do
+      # Allowing underscores is a feature of the `\w` regex.
```

Comment:
Is that better? I've amended one of the tests, too, to test that it strips all but one non-trailing space character, as per `/[^a-z0-9 ]/i`.